src/components/taskDetails/TaskHeader.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { Button, Textarea } from './index';
+import styles from './task-details.module.scss';
+
+interface TaskHeaderProps {
+    isEditing: boolean;
```

Comment:
again the name could be isEditingMode

Reply:
if I changed it to `isEditingMode` I have to changed it into many files not just here